### PR TITLE
Style events like a blockchain

### PR DESCRIPTION
**Pull request by vluna**
## Description
- Update overall style for the events
- Update `EventCard` component with new style for the events
- Update `UpcomingEventsList` component with new changes

## Related Issue
#4714

### CODE REVIEW

**Comment by vluna** [comment]
@minimalsm I am not super familiar with gatsby/react, so please let me know if there is a better way 🙂

File: src/components/EventCard.js
Comment (wackerow) [inline src/components/EventCard.js:126]: Couple thoughts here on ways to reduce redundant code...
1- Pass `isEven` as a prop to `StyledCardContent` (`<StyledCardContent isEven={isEven}>`) and then pick this up in the CSS to conditionally render on the left or right.
2- Could try using `&:nth-child(even)` from the parent to style all even children differently. 
3- Could otherwise declare a constant before the return in this component that holds all the children that are currently written twice, then reuse that constant (as a mini component) in the final return. 

The CSS approach I think would be the cleanest, but otherwise the first suggestion would suffice and would remove the need to have this duplicated.

File: src/components/UpcomingEventsList.js
Comment (wackerow) [inline src/components/UpcomingEventsList.js:116]: If we go with the approach on `EventCard` of passing `isEven` as a prop, you could calculate that here, naming it the same to make it easy to track, and then you could remove needing to declare this as a constant in the EventCard component. 

 ```suggestion
              isEven={idx % 2 === 0}
```

**Review by wackerow — CHANGES_REQUESTED** [review]
@vluna Very cool! Thanks for your work on this so far. 

Left a couple comments around trying to minimize code redundancy. Also, getting some rough results on mobile sizes. We'll need to fix this up to be mobile responsive. 

image

I would suggest this switches to just a vertical string of boxes with vertical lines in the middle connecting them. Open to thoughts from others though.

**Comment by vluna** [comment]
> @vluna Very cool! Thanks for your work on this so far.
> 
> Left a couple comments around trying to minimize code redundancy. Also, getting some rough results on mobile sizes. We'll need to fix this up to be mobile responsive.
> 
> image
> 
> I would suggest this switches to just a vertical string of boxes with vertical lines in the middle connecting them. Open to thoughts from others though.

Thanks for pointing that out. I guess the responsiveness slipped my mind.

File: src/components/EventCard.js
Comment (vluna) [reply src/components/EventCard.js:126]: Nice! Thanks for the suggestions, will work on them 🙂

**Comment by vluna** [comment]
@wackerow Thanks for the feedback and suggestions. I have updated the events page to be mobile friendly.

@minimalsm Let me know what you think 🙂

**Comment by minimalsm** [comment]
Hey @vluna,

Thanks a ton for this. Apologies for the lack of image reference on this. I think this could use some design tweaking but definitely beyond our capacity until we've got a designer on staff—you've done an awesome job here. Merging now ❤️❤️❤️❤️❤️❤️❤️❤️

**Comment by minimalsm** [comment]
Just waiting on build after resolving merge conflicts here :-)

**Review by minimalsm — APPROVED** [review]

**Pull request merged**
